electron-app/src/py-handler.ts

```diff
@@ -174,7 +179,7 @@ export default class PyHandler {
     // start of the path
     if (process.env.VIRTUAL_ENV) {
       process.env.PATH =
-        process.env.VIRTUAL_ENV + path.sep + 'bin:' + process.env.PATH;
+        process.env.VIRTUAL_ENV + path.sep + (process.platform === 'win32' ? 'Scripts;' : 'bin:') + process.env.PATH;
```

Comment:
@kelsos @isidorosp can you also check if this makes sense in Windows? Other than that looks fine to me.

Reply:
I can confirm in the virtual environment I am using in Windows the folder name is `Scripts` and not `bin`. To activate the virtual environment I have to run a `venv\Scripts\activate.bat`.

Reply:
All right. Thanks